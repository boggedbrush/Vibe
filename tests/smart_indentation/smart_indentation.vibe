# VibeSpec: 1.6
patch_type: add_block
file: hello.py
position: before
anchor: '^\s*print\("--- Original Table ---"\)' # Regex for the target print line
--- code: |
  # This is a test of smart indentation.
  # This block is written at column 0.
  # Vibe CLI should re-indent it based on the anchor.
  test_variable_smart_indent = "It worked!"
  if True:
      print(f"Smart Indent Test: {test_variable_smart_indent}")
  # End of smart indent test block.