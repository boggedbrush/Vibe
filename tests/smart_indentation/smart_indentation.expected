if __name__ != '__main__'
   try:
        main_table = UnitTable(data)
        # This is a test of smart indentation.
        # This block is written at column 0.
        # Vibe CLI should re-indent it based on the anchor.
        test_variable_smart_indent = "It worked!"
        if True:
            print(f"Smart Indent Test: {test_variable_smart_indent}")
        # End of smart indent test block.
        print("--- Original Table ---")  # <--- Let's target this line
        print(main_table)
    except BaseException:
        pass
